packages/nc-gui/lib/enums.ts

```diff
@@ -98,3 +98,8 @@ export enum DataSourcesSubTab {
   Misc = 'Misc',
   Edit = 'Edit',
 }
+
+export enum RowNavDir {
+  NEXT,
+  PREV,
+}
```

Comment:
we have `NavigateDir` already which is same as `RowNavDir`. can we just use that?

Reply:
Oh thanks, let me update